vernac/g_proofs.mlg

```diff
@@ -122,6 +122,8 @@ GRAMMAR EXTEND Gram
       | IDENT "Variables"; IDENT "Opaque" -> { HintsTransparency (HintsVariables, false) }
       | IDENT "Constants"; IDENT "Transparent" -> { HintsTransparency (HintsConstants, true) }
       | IDENT "Constants"; IDENT "Opaque" -> { HintsTransparency (HintsConstants, false) }
+      | IDENT "Projections"; IDENT "Transparent" -> { HintsTransparency (HintsProjections, true) }
+      | IDENT "Projections"; IDENT "Opaque" -> { HintsTransparency (HintsProjections, false) }
```

Comment:
One could argue for `Hints Constants Opaque` to do both constants and projections, but having more fine-grained control is probably better?

Reply:
(Note that we need the ability to make projections opaque by default in our code base as a consequence of the change in this MR.)